articles/azure/azs-how-create-vm-image-portal.md

```diff
@@ -0,0 +1,121 @@
+---
+title: How to create a custom image from a VM on Azure Stack using the UKCloud Azure Stack portal | UKCloud Ltd
+description: Provides details on how to create a custom image from a VM using the portal
+services: azure-stack
+author: Daniel Brennand
+reviewer:
+lastreviewed:
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create a custom image from a VM using the UKCloud Azure Stack portal
```

Comment:
use the - Portal style here to keep the title shorter as it is displayed in an area with less space:
Create a customer image from a VM - Portal

Reply:
Still needs changing per Sue's comment